Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -776,30 +777,21 @@ - (NSDictionary *)getBiddingDataWithPlacementId:(NSString *)placementId {
     return @{@"token": returnedToken};
 }
 
-- (BannerSize)getBannerSize:(ISBannerSize *)size {
-    if ([size.sizeDescription isEqualToString:@"RECTANGLE"]) {
-        return BannerSizeMrec;
-    } else if ([size.sizeDescription isEqualToString:@"SMART"]) {
+- (VungleAdSize *)getBannerSize:(ISBannerSize *)size {
+    if ([size.sizeDescription isEqualToString:kSizeCustom]) {
+        return [VungleAdSize VungleAdSizeFromCGSize:CGSizeMake(size.width, size.height)];
+    } else if ([size.sizeDescription isEqualToString:kSizeBanner] || [size.sizeDescription isEqualToString:kSizeLarge]) {
```

Comment:
u dont need this els if block cause u return regular as default. @ashinagawa

Reply:
Updated.